packages/turbo-codemod/src/transforms/add-default-outputs.ts

```diff
@@ -0,0 +1,53 @@
+import { Flags } from "../types";
+import path from "path";
+import fs from "fs-extra";
+import { error, ok, skip } from "../logger";
+
+const DEFAULT_OUTPUTS = ["dist/**", "build/**"];
+
+interface TaskDefinition {
+  outputs: [];
+}
+
+interface PipelineConfig {
+  [taskName: string]: TaskDefinition;
+}
+
+export default function addDefaultOutputs(files: string[], flags: Flags) {
+  // We should only get a directory as input
+  if (files.length !== 1) {
+    return;
+  }
+
+  // TODO: handle config in package.json turbo keys?
```

Comment:
No. You should run that migration first.

Reply:
you mean in this? or we expect the user to run that first?